backend/src/utils/ffmpeg_utils.py

```diff
@@ -454,21 +461,40 @@ def find_filters(video_path: str) -> List[Filters]:
     return global_filters
 
 
-def find_transi(video_path: str) -> Transition:
+def find_transi(video_path: str) -> Tuple[Transition, Transition]:
     """
-    Find the transition for the video and return its name and the time it last
+    Find the starting and ending transition for the video
     """
-    TRANSI = get_transitions()
-    global_transi: Tuple[str, int] = ("", 0)
-    if "transi" in TRANSI:
-        global_transi = TRANSI["transi"]
+    TRANSI = get_transition()
+    in_transi: Tuple[str, Union[None, Transition]] = ("", None)
+    out_transi: Tuple[str, Union[None, Transition]] = ("", None)
```

Comment:
That would be nice to use [named tupple](https://docs.python.org/3/library/collections.html#collections.namedtuple) for later usage